packages/ckeditor5-image/src/imageutils.ts

```diff
@@ -71,7 +71,8 @@ export default class ImageUtils extends Plugin {
 	}
 
 	/**
-	 * Handles inserting single file. This method unifies image insertion using {@link module:widget/utils~findOptimalInsertionRange}
+	 * Handles inserting single file. This method unifies image insertion
+	 * using {@link module:engine/model/schema~Schema#findOptimalInsertionRange}
```

Comment:
This was using the widget implementation of this method. I think this should not change.

Reply:
Adressed in 24ac1fa